packages/blocks/src/api/factory.js

```diff
@@ -313,30 +378,34 @@ export function getBlockTransforms( direction, blockTypeOrName ) {
  * @param {Array|Object} blocks Blocks array or block object.
  * @param {string}       name   Block name.
  *
- * @return {Array} Array of blocks.
+ * @return {?Array} Array of blocks or null.
  */
 export function switchToBlockType( blocks, name ) {
 	const blocksArray = castArray( blocks );
-	const isMultiBlock = blocksArray.length > 1;
+	const isMultiBlock = isMultiBlockSelection( blocksArray );
```

Comment:
Minor: The terminology of a "selection" should not have meaning in this module, as it is specific to the implementation of an editor UI, not of blocks data. For what it's worth, I don't think a separate `isMultiBlockSelection` function provides a ton of value over how this was written previously.

Reply:
> The terminology of a "selection" should not have meaning in this module

So it's the term / word rather than the actual test being conducted here that is of concern?

I think the reason I create the function was because I was refactoring out a function that had too many responsibilities into 2 separate functions. In hindsight, the `isMultiBlockSelection` function was just redundant.

Reply:
Fixed

Reply:
> So it's the term / word rather than the actual test being conducted here that is of concern?

Yes on the terminology, though it only became relevant by the introduction of a function, which in my head prompted the question for why it should exist at all.